platform-sdk/swirlds-platform-core/src/test/java/com/swirlds/platform/reconnect/emergency/EmergencyReconnectTests.java

```diff
@@ -228,7 +229,7 @@ private ReconnectController createReconnectController(
                 new ReconnectLearnerFactory(
                         getStaticThreadManager(),
                         addressBook,
-                        mock(ReconnectSettings.class),
+                        ConfigurationHolder.getConfigData(ReconnectConfig.class),
```

Comment:
I assume with some refactoring we can get rid of that `ConfigurationHolder` call.